var/spack/repos/builtin/packages/octave/package.py

```diff
@@ -270,6 +270,20 @@ def configure_args(self):
         else:
             config_args.append("--without-z")
 
+        if '^mkl' in spec and 'gfortran' in self.compiler.fc:
```

Comment:
Wouldn't it be better to move this up to where these options are set first, around line 160?

Reply:
Yes, please. I followed the https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/r/package.py, but should have seen the BLAS config around line 160 in this package file.

Reply:
It took me some trying to get the formatting style right but it's looking OK now. I am not sure what's up with the failed codecov/project check, anyone knows how to fix that?